packages/datadog-plugin-http/src/client.js

```diff
@@ -83,22 +83,14 @@ function patch (http, methodName, tracer, config) {
 
             scope.bind(res)
 
-            span.setTag(HTTP_STATUS_CODE, res.statusCode)
-
-            addResponseHeaders(res, span, config)
-
-            if (!config.validateStatus(res.statusCode)) {
-              span.setTag('error', 1)
-            }
-
             res.on('end', () => finish(req, res, span, config))
 
             break
           }
           case 'error':
             addError(span, arg)
           case 'abort': // eslint-disable-line no-fallthrough
-          case 'close': // eslint-disable-line no-fallthrough
+          case 'timeout': // eslint-disable-line no-fallthrough
```

Comment:
so we just ignore close now?

Reply:
I couldn't find a flow that doesn't have one of the other events before `close`.

Reply:
Yeah that sounds about right.